lib/py/src/transport/TSocket.py

```diff
@@ -183,7 +187,7 @@ def listen(self):
         if hasattr(self.handle, 'settimeout'):
             self.handle.settimeout(None)
         self.handle.bind(res[4])
-        self.handle.listen(128)
+        self.handle.listen(self._backlog)
```

Comment:
To preserve the existing behavior, how about `self.handle.listen(self._backlog or 128)`, in combination with a default value of None in the class constructor, this eliminates the need for setBacklog().

Reply:
We shoud not use `self.handle.listen(self._backlog or 128)` to preserve the existing behavior, because the backlog parameter accept a zero value.
Plz have a look at the python source code below:
```c
// From Python-2.7.14/Modules/socketmodule.c
/* s.listen(n) method */

static PyObject *
sock_listen(PySocketSockObject *s, PyObject *arg)
{
    // some code before
    /* To avoid problems on systems that don't allow a negative backlog
     * (which doesn't make sense anyway) we force a minimum value of 0. */
    if (backlog < 0)
        backlog = 0;
    res = listen(s->sock_fd, backlog);
    // some code more
}
```

I have think about `self._backlog if self._backlog is None else 128 `, and the code seems to be redundant with bad smell. So declaring the _backlog attribute with a default value of 128 in the class constructor seems to be a better way, isn't? On the other hand, `__init__` will be called before listen, while the listen method called, `this.handle.listen` will receive a `_backlog` with a default value of 128,  existing behavior preserved!